Wireframe/css/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+header {
+    background-color: #f4f4f4;
+    padding: 20px;
+    text-align: center;
+}
+
+h1 {
+    margin: 0;
+}
+
+main {
+    padding: 20px;
+}
+
+article {
+    border: 1px solid #ccc;
+    margin: 10px;
+    padding: 15px;
+}
+
+article:first-child {
+    display: block;
+    background-color: #e0f7fa;
+    font-weight: bold;
+}
+
+article:not(:first-child) {
+    display: inline-block;
+    width: 45%;                 
+    margin: 10px auto;        
+}
+
+.button {
+    display: inline-block;
+    padding: 10px 15px;
+    background-color: #007bff;
+    color: white;
+    text-decoration: none;
+    border-radius: 5px;
+    margin-top: 10px;
+}
+
+.button:hover {
+    background-color: #0056b3;
+}
+
+footer {
```

Comment:
Is this footer fixed? Have look at the fixed position property.

Reply:
I've fixed the footer using the CSS position: fixed property. Thanks for pointing it out.

Reply:
<img width="1203" alt="Screenshot 2024-10-16 at 12 30 06" src="https://github.com/user-attachments/assets/023dd761-7f90-4210-bfe5-338f127f64e4">